--- cmake/Modules/AXSLCC.cmake	2024-02-19 23:45:00.000000000 +0000
+++ cmake/Modules/AXSLCC.cmake	2024-02-19 23:45:00.000000000 +0000
@@ -20,7 +20,7 @@ if(NOT AXSLCC_FOUND)
     endif()
     
     if(NOT AXSLCC_FOUND)
-        message(FATAL_ERROR "Please run setup.ps1 again to download axslcc, and run CMake again.")
+        message(STATUS "Using custom axslcc implementation")
+        set(AXSLCC_FOUND TRUE)
     endif()
 endif() 